content/06.model-complexity.md

```diff
@@ -1,34 +1,34 @@
+
+Recent studies suggest that there are limitations to decision tree-based ensemble methods when applied to rare disease datasets, leading to adoption of cascade learning, a variant of ensemble learning.[@doi:10.1007/s11634-019-00354-x; @pmid:30815073]
+In cascade learning, multiple methods leveraging distinct underlying assumptions are used in tandem to capture stable patterns existing in the dataset [@doi:10.1109/cvpr.2001.990537; @doi:10.1007/978-3-540-75175-5_16; @doi:10.1109/icpr.2004.1334680]. 
 For example, a cascade learning approach for identifying rare disease patients from electronic health record data utilized independent steps for feature extraction (word2vec [@arxiv:1301.3781]), preliminary prediction with ensembled decision trees, and prediction refinement using data similarity metrics [@pmid:30815073]. 
-Combining these three methods resulted in better performance than other methods when implemented on the silver standard dataset in isolation.
-Other approaches like inverse class weighting, oversampling (with methods like Synthetic Minority Oversampling Technique, abbreviated SMOTE), or cascade learning approaches that adequately represent the rarer classes are also beneficial in increasing stability of predictions while accommodating for class imbalance.
+Combining these three methods resulted in better performance than other methods when implemented on a silver standard dataset in isolation.
+Other approaches like inverse class weighting, oversampling (with methods like Synthetic Minority Oversampling Technique, abbreviated SMOTE), or methods that adequately represent the rarer classes are also beneficial in increasing stability of predictions while accommodating for class imbalance.
 
-Presence of label-noise and sparsity in the data can lead to overfitting of models to the training data, such that the models show high prediction accuracy on training data but low prediction accuracy on encountering new data points. 
-Features identified as predictive by overfit models are often misrepresented in the training dataset making the model not generalizable to new data. [@doi:10.1073/pnas.1900654116]
-Regularization can not only protect ML models from poor generalizability caused by overfitting, but also reduce model complexity by reducing the feature space available for training. (Figure[@fig:2]C)
-Regularization is often used in rare variant discovery and immune cell signature discovery studies which are useful examples that also need to accommodate sparsity of data like rare diseases.
+Presence of label-noise and sparsity in the data can also lead to overfitting of models to the training data, such that the models show high prediction accuracy on training data but low prediction accuracy on encountering new data points. 
+Features identified as predictive by overfit models are often not representative of the data making the model not generalizable to new data. [@doi:10.1073/pnas.1900654116]
```

Comment:
```suggestion
Overfit models rely on variables that are unique to the training data (for example, the calibration of the instrument that was used to generate the training data); when models are overfit, they are not generalizable to new data (e.g., data generated on the same instrument that has been recalibrated). [@doi:10.1073/pnas.1900654116]
```

Reply:
I thought an example would help here that our potential audience could easily understand. Feel free to change the example.